packages/squiggle-lang/__tests__/helpers/reducerHelpers.ts

```diff
@@ -72,8 +72,13 @@ export function testEvalToBe(expr: string, answer: string, only = false) {
     : test(expr, async () => await expectEvalToBe(expr, answer));
 }
 
-export function testEvalToMatch(expr: string, expected: string | RegExp) {
-  test(expr, async () => await expectEvalToMatch(expr, expected));
+export function testEvalToMatch(
+  expr: string,
+  expected: string | RegExp,
+  only = false
```

Comment:
Reminder that `only` is dangerous (too easy to add it and forget to remove), and `jest -t "name pattern"` is usually better.